doc/source/notebooks/manual/advanced/multiclass_classification.ipynb

```diff
@@ -4,10 +4,424 @@
+      "text/plain": [
+       "<Figure size 864x432 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.figure(figsize=(12, 6))\n",
+    "order = np.argsort(X.reshape(-1,))\n",
+    "colors = plt.cm.winter(np.linspace(0, 1, C))\n",
+    "for c in range(C):  \n",
+    "    plt.plot(X[order], f[order, c], '.', color=colors[c], label='%d'%c);\n",
+    "    plt.plot(X[order], Y_hot[order, c], '-', color=colors[c])\n",
+    "plt.legend()\n",
+    "plt.xlabel('$X$')\n",
+    "plt.ylabel('latent (dots) and one-hot labels (lines)')\n",
+    "plt.title('Sample from the joint $p(Y, \\mathbf{f})$')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Inference\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Inference here consists in computing the posterior distribution over the latent functions given the data $p(\\mathbf{f}|Y, X)$.\n",
+    "\n",
+    "Different inference methods may be used, including\n",
+    " * MCMC sampling\n",
```

Comment:
We had the impression you would explain both methods here. Could you link to the MCMC notebook here please instead?

Reply:
made it more clear